migrate/20240516_boot_image_size_gib.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:boot_image) do
+      add_column :size_gib, Integer, default: 0
```

Comment:
Why don't we set `default: nil` initially? I didn't get why do we set it as `0`?